.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,12 +23,20 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop (please complete the following information):**
- - OS: [e.g. iOS]
- - npm version:
- - node version:
- - electron verison:
- - electron-forge version:
+**Please complete the following information:**
+<!-- 1. navigate to help -> about -->
+<!-- 2. press copy to copy the full version info to your clipboard -->
+<!-- 3. paste version in here -->
+```
+Version:
```

Comment:
Is the formatting intended compared to before

Reply:
yes, I just copied it from the help > about tab.

Reply:
Ok